test/LedSign.js

```diff
@@ -107,87 +107,50 @@ describe('LedSign', () => {
         .to.equal(VALID_SIGN_REQUEST.backgroundColor);
       expect(signResponse.textColor).to.equal(VALID_SIGN_REQUEST.textColor);
       expect(signResponse.borderColor).to.equal(VALID_SIGN_REQUEST.borderColor);
-      done();
     });
-    it('Should return statusCode 404 when the sign is down', done => {
+    it('Should return statusCode 404 when the sign is down', async () => {
       healthCheckMock.rejects(ERROR_MESSAGE);
-      chai
```

Comment:
just curious, is the point of this refactor to get rid of chai and implement our own testing scheme

Reply:
the refactor decreases the number of lines used for each test. We still call chai, just in another class

Reply:
ok gotchu that makes sense